ec2/sc-product-ec2-linux-jumpcloud-notebook.yaml

```diff
@@ -41,6 +37,7 @@ Resources:
       'Fn::Transform':
         Name: 'AWS::Include'
         Parameters:
+          # source: https://github.com/Sage-Bionetworks/admincentral-infra/blob/master/templates/cfn-snippets-bucket.yaml
```

Comment:
Why comment out rather than remove?

Reply:
this is not a CFN property, although i can see why you would think it is.    it's just a comment to indicate the source of the snippet otherwise we may not remember where to look to make updates.

Reply:
Oh, I see. Thanks for the clarification.